themes/berbera/layouts/index.html

```diff
@@ -56,15 +56,10 @@ <h4 class="card-header" id="colab_example">Try now on Colab</h4>
         <div class="row">
             <div class="col-sm-12">
                 <div class="card">
-                    <h4 class="card-header" id="hsfs_basics">Basics</h4>
+                    <h4 class="card-header" id="hsfs_basics">Create your first Features</h4>
                     <div class="card-body">
                         <ul>
-                            {{ range (where .Pages "File.Dir" "in" "/featurestore/hsfs/basics/").ByTitle }}
-                            <li>
-                                <a href="{{.Permalink}}">{{.Title}}</a>
-                            </li>
-                            {{ end }}
-                            {{ range (where .Pages "File.Dir" "in" "/featurestore/datasets/").ByTitle }}
+                            {{ range (where .Pages "File.Dir" "in" "/featurestore/hsfs/basics/feature_engineering/").ByTitle }}
```

Comment:
Isn't it `{{ range (where .Pages "File.Dir" "in" "/featurestore/hsfs/basics/").ByTitle }}` ?